### ISSUE TITLE
Robust statistical analyses by Wilcox

### ISSUE THREAD

**Issue by kpriftis** [jasp-stats/jasp-desktop#1381]
Hi there and congratulations for JASP! It is wonderful! I already use it with my undergraduate students.

How about considering to implement in JASP,  robust analyses developed by Rand Wilcox (http://dornsife.usc.edu/labs/rwilcox/software/)? Robust analyses are extremely useful in psychological research, where in many cases the assumptions of classic parametric tests are violated. These analyses are already available in R: https://cran.r-project.org/web/packages/WRS2/index.html

All the very best,
Konstantinos Priftis

**Comment by Suggestions-Only** [comment]
Since we're discussing robust analysis, one thing I would suggest implementing is an option for a Welch ANOVA and Games-Howell post hoc tests. They seem to be a little more powerful than comparing medians, while still avoiding common linear model assumptions. Also, including an option for MM-estimation for linear regression would be great.

**Comment by Suggestions-Only** [comment]
And I've also found a Games-Howell R function on GitHub that someone else has written.  It looks like it could be written into the R code of JASP rather easily, although I might be mistaken. 

https://gist.github.com/aschleg/ea7942efc6108aedfa9ec98aeb6c2096

**Comment by boutinb** [comment]
@EJWagenmakers Can you give an answer to this request?

**Comment by EJWagenmakers** [comment]
@JohnnyDoorn 
Yes these are very useful remarks. We should add a module for robust analyses to our JASP to-do list. And the other suggestions are useful too. Let me add it on the JASP whiteboard.
E.J.
